src/applications/edu-benefits/0994/containers/ConfirmationPage.jsx

```diff
@@ -60,6 +61,21 @@ class ConfirmationPage extends React.Component {
           <br />
           <i>Please print this page for your records.</i>
         </p>
+        {!_.get(form.data, 'appliedForVaEducationBenefits', true) && (
```

Comment:
since this is also done line 110 maybe pull this out into a variable

Reply:
Change has been applied.